docs/conf.py

```diff
@@ -58,9 +73,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.1'
+version = '.'.join(meta['version'].split('.')[:2])
```

Comment:
Maybe it should be documented in `letsencrypt/__init__.py` that `__version__` must be of such form (`X.Y` at least)?